dlp/snippets/inspect/inspect_test.go

```diff
@@ -500,3 +505,23 @@ func TestInspectImageFile(t *testing.T) {
 		t.Errorf("TestInspectImageFile got %q, want %q", got, want)
 	}
 }
+
+func TestInspectImageFileListedInfoTypes(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+	pathToImage := "testdata/test.png"
+
+	if err := inspectImageFileListedInfoTypes(&buf, tc.ProjectID, pathToImage); err != nil {
+		t.Fatal(err)
+	}
+	got := buf.String()
+	if want := "Info type: PHONE_NUMBER"; !strings.Contains(got, want) {
+		t.Errorf("inspectImageFileListedInfoTypes got %q, want %q", got, want)
+	}
+	if want := "Info type: EMAIL_ADDRESS"; !strings.Contains(got, want) {
+		t.Errorf("inspectImageFileListedInfoTypes got %q, want %q", got, want)
+	}
+	if want := "Info type: US_SOCIAL_SECURITY_NUMBER"; strings.Contains(got, want) {
```

Comment:
The test.png file https://github.com/GoogleCloudPlatform/golang-samples/blob/main/dlp/snippets/inspect/testdata/test.png
does not contain SSN. Am I missing something? Do the tests pass?
Do we need another image file?

Reply:
Updated it.